frontend/Docker/nginx.conf

```diff
@@ -36,12 +36,19 @@ http {
         resolver 127.0.0.11 ipv6=off valid=10s;
```

Comment:
is this nginx file only used for local development?
If so again to previous comment about not having a frontend backend in the docker-compose file for local development.  Pretty sure we already have an nginx file for both the front end and back end